### ISSUE TITLE
Add more info to README about building and running project

### ISSUE THREAD

**Issue by andrewmarklloyd** [SoftwareEngineeringDaily/SEDaily-Android#127]
We should add more information about how to build and run the project including software installation and dependencies

**Comment by TheHollidayInn** [comment]
Can you be more specific? Would you just like a line saying to use Android studio?

**Comment by andrewmarklloyd** [comment]
Yeah that would be good. I was just thinking for someone who hasn't done any Android development who is wanting to get into it they might not know where to get started.

**Comment by andrewmarklloyd** [comment]
I am thinking we should standardize the README's across all projects. I can look through the README's and come up with a structure to follow for all of them if you want.
